src/annotator/components/NotebookModal.js

```diff
@@ -0,0 +1,96 @@
+import { createSidebarConfig } from '../config/sidebar';
```

Comment:
Can you sort the local imports after external imports. The ordering should be:

```
<builtin or Node module imports>

<3rd-party module imports>

<local imports>
```

Within each group, sort by module path. Note that not all existing code perfectly adheres to this sort order.

Reply:
I will.